msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CompileSceneKitAssetsTaskBase.cs

```diff
@@ -122,6 +125,7 @@ int CopySceneKitAssets (string scnassets, string output, string intermediate)
 				args.AddQuotedFormat ("--target-version-{0}={1}", OperatingSystem, SdkVersion);
 			}
 			args.AddQuotedFormat ("--target-build-dir={0}", Path.GetFullPath (intermediate));
+			args.AddQuotedFormat ("--resources-folder-path={0}.app", AppBundleName);
 
```

Comment:
Hmmm, you should probably use AppBundleDir and then get the Path.GetFIleName() so that you have the appropriate .app or .appex extension.

Reply:
Good point! I've fixed it now.

Reply:
AppBundlePath ends with '/', but AppBundleDir does not. Both will always be defined.

Reply:
> AppBundlePath ends with '/', but AppBundleDir does not

That's a subtle difference :smile: I've changed it to use AppBundleDir.

> Both will always be defined.

I had to depend on `_GenerateBundleName`, otherwise things stopped working:

> /work/maccore/master/xamarin-macios/_mac-build/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/Mac/Xamarin.Mac.Common.targets: error : Error executing task CompileSceneKitAssets: Required property 'AppBundleDir' not set.

(same for Xamarin.iOS as well)